js_upload/js_upload.php

```diff
@@ -333,7 +334,10 @@ function handleUpload(){
         $pathinfo = pathinfo($this->file->getName());
         $filename = $pathinfo['filename'];
 
-        $ext = $pathinfo['extension'];
+        if (!isset($pathinfo['extension'])) {
+        	Logger::warning('extension isn\'t set.', ['filename' => $filename]);
+		}
```

Comment:
Code standard: Mixed indentation.

Reply:
Fixed the indentation for the whole file ;)